src/apis/club-posts/services/club-posts.service.ts

```diff
@@ -136,9 +141,29 @@ export class ClubPostsService {
     ];
   }
 
-  async isExistOrNotFound(postId: number): Promise<true> {
+  async findOneOrNotFound(
+    clubId: number,
+    postId: number,
+    overrideOptions?: FindOneOptions<ClubPost>,
+  ): Promise<ClubPostDto> {
+    const existPost = await this.clubPostRepository.findOne({
+      where: { id: postId, clubId, status: ClubPostStatus.Posting },
+      relations: { user: true },
+      ...overrideOptions,
+    });
```

Comment:
```suggestion
    overrideOptions?: FindOneOptions<ClubPost> = {},
  ): Promise<ClubPostDto> {
    const existPost = await this.clubPostRepository.findOne({
      where: { id: postId, clubId, status: ClubPostStatus.Posting },
      relations: { user: true },
      ...overrideOptions,
    });
```

undefined는 이터러블 객체가 아니기때문에 이런 방어코드가 필요할듯합니다.